Assets/Scripts/Player/PlayerGroundCheck.cs

```diff
@@ -0,0 +1,32 @@
+﻿using UnityEngine;
+using System.Collections;
+
+public class PlayerGroundCheck : MonoBehaviour
+{
+    [SerializeField] private Transform m_groundCheckTF;
+    [SerializeField] private float m_groundCheckRadius;
+    [SerializeField] private LayerMask m_groundLayer;
+
+    public bool IsGrounded()
+    {
+        Collider[] hits = new Collider[5];
+        Physics.OverlapSphereNonAlloc(m_groundCheckTF.position, m_groundCheckRadius, hits, m_groundLayer);
+
```

Comment:
In here is where it should call the GizmosUtility.DrawWireSphere(Layer, sphereParams).
Sphere params should include sphere position, rotation, color etc.
Layer should be one of GizmosUtility layers, most likely player movement layer in this case.

Make sure to wrap the GizmosUtility.DrawWireSphere call with #if UNITY_EDITOR #endif